File: C++/1123.cpp
#include <iostream>
using namespace std;


int main()
{
    int x1, y1, x2, y2, dx, dy;

    cin >> x1 >> y1 >> x2 >> y2;

    dx = x1 - x2;
    if (dx < 0)
        dx = -dx;
    dy = y1 - y2;
    if (dy < 0)
        dy = -dy;

    if ((dx == 1 && dy == 2) || (dx == 2 && dy == 1))
        cout << "YES" << endl;
    else
        cout << "NO" << endl;

    return 0;
}

File: Python/1123.py
x1, y1 = map(int, input().split())
x2, y2 = map(int, input().split())

dx = abs(x1 - x2)
dy = abs(y1 - y2)

if dx == 2 and dy == 1 or dx == 1 and dy == 2:
    print('YES')
else:
    print('NO')
